src/main/java/hudson/plugins/jacoco/model/CoverageObject.java

```diff
@@ -203,7 +204,7 @@ public Coverage getLineCoverage() {
 	/**
 	 * Gets the build object that owns the whole coverage report tree.
 	 */
-	public abstract AbstractBuild<?,?> getBuild();
+	public abstract Run<?,?> getBuild();
```

Comment:
:+1: to break API is possible (BTW it's useful to search in GitHub under user:jenkinsci before breaking it)